test/ctia/flows/hooks/redismq_event_hook_test.clj

```diff
@@ -25,12 +25,13 @@
                                     es-helpers/fixture-properties:es-store
                                     fixture-properties:redismq-hook
                                     test-helpers/fixture-properties:events-enabled
-                                    test-helpers/fixture-ctia
-                                    test-helpers/fixture-allow-all-auth]))
+                                    test-helpers/fixture-allow-all-auth
+                                    test-helpers/fixture-ctia]))
```

Comment:
Prep: `fixture-ctia` will be in charge of starting an `app`, so it must be last in order to pick up other fixtures in the ConfigService.